### DISCUSSION TITLE
How to push data to Github Desktop?

### DISCUSSION THREAD

**Discussion by victoriaakelly** [community/community#23766] [New to GitHub]
<p>Maybe I’m an idiot…I feel like I’ve read every post online and asked everyone around me how to do this, but I can’t figure it out. How do you push data you’ve collected to Github using Github Desktop?? I have it set up on my desktop and can navigate around the app. I’m not sure how to get the data I’ve collected and push it to a specific project on Github. Any help?</p>

**Comment by steveward** [comment, marked as answer]
<p><a class="mention" href="/victoriaakelly">@victoriaakelly</a> great question! Our training team put together a <a href="https://www.youtube.com/watch?v=C69-s2o9wqw" rel="nofollow noopener">tutorial video</a> that may be helpful for you. We also have some GitHub Desktop <a href="https://help.github.com/desktop/guides/" rel="nofollow noopener">help documentation</a> available for you to reference. Here’s a basic workflow using GitHub Desktop:</p>
<ol>
<li>
<p>Make changes in your project using your IDE/text editor</p>
</li>
<li>
<p>Switch back to GitHub Desktop – you should see changed files in the <code>Changes</code> tab in the left sidebar</p>
</li>
<li>
<p>Commit those changes in the bottom-left corner of GitHub Desktop. Make sure to enter a commit summary!</p>
</li>
<li>
<p>Click the <code>Push origin</code> button to push the changes up to the remote repository on GitHub. If you have not yet created a repository on GitHub you will see a <code>Publish repository</code> option that will allow you to create the remote repository.</p>
</li>
</ol>
<p>You’ve now pushed your changes up to GitHub. If you encoutner any errors, please let me know! I would also recommend reading through some of our <a href="https://help.github.com/articles/git-and-github-learning-resources/" rel="nofollow noopener">recommended resources</a> for getting started with Git and GitHub. I hope that helps!</p>

**Comment by ol3h-unag1** [comment]
<p>In my case I should open in editor via Git Desktop menu, in this case Git Desktop will be able to track changes in em.<br>
<img src="https://user-images.githubusercontent.com/32937411/181079260-20e16caa-288e-4950-8bd3-5a516c6df66a.png" alt="image" data-base62-sha1="Aj4RuC2SVOoGAw7O2rmxedc2Eh" width="662" height="340"></p>
<p>P.S. Git will track only changes in tab of the editor that was triggered via Git’s “open in Notepad” or “open with default program” actions.</p>
